apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -607,11 +607,9 @@ private void loadSample(DataSource dataSource, List<StaticRifResource> sampleRes
 
     // Verify that the expected number of records were run successfully.
     assertEquals(0, failureCount.get());
-    assertEquals(
-        sampleResources.stream().mapToInt(r -> r.getRecordCount()).sum(),
-        loadCount.get(),
-        "Unexpected number of loaded records.");
-
+    assertTrue(
+        (loadCount.get() >= sampleResources.stream().mapToInt(r -> r.getRecordCount()).sum()),
+        "The number of loaded records should meet or exceed the number of expected records. The expected records can be exceeded because of line items.");
```

Comment:
can we not adjust the counts to account for the line items so that we can have a more precise assertion?